plugins/modules/ocapi_command.py

```diff
@@ -0,0 +1,268 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2022 Western Digital Corporation
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: ocapi_command
+short_description: Manages Out-Of-Band controllers using Open Composable API (OCAPI).
+description:
+  - Builds OCAPI URIs locally and sends them to remote OOB controllers to
+    perform an action.
+  - Manages OOB controller such as Indicator LED, Reboot, Power Mode, Firmware Update.
+options:
+  category:
+    required: true
+    description:
+      - Category to execute on OOB controller.
+    type: str
+  command:
+    required: true
+    description:
+      - Command to execute on OOB controller.
+    type: str
+  baseuri:
+    required: true
+    description:
+      - Base URI of OOB controller.
+    type: str
+  proxy_slot_number:
+    description: For proxied inband requests, the slot number of the IOM.  Only applies if baseuri is a proxy server.
```

Comment:
```suggestion
    description: For proxied inband requests, the slot number of the IOM.  Only applies if I(baseuri) is a proxy server.
```
See https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation